### PR TITLE
Some maintenance things

### PR DESCRIPTION
**Pull request by JayH5**
* ~~Try build uWSGI on Alpine again~~
* Don't build cffi for PyPy, should use bundled version
* Do build psycopg2 binaries for all supported platforms (http://initd.org/psycopg/articles/2018/02/08/psycopg-274-released/)

### CODE REVIEW

**Comment by JayH5** [comment]
Not done yet--I've broken caching

File: requirements-debian.txt [hunk 1]
```diff
@@ -4,8 +4,6 @@
 uWSGI==2.0.16
 
 # These have pre-compiled manylinux packages on CPython but not on PyPy
-cffi==1.11.4; platform_python_implementation!='CPython'
 cryptography==2.1.4; platform_python_implementation!='CPython'
 Pillow==5.0.0; platform_python_implementation!='CPython'
-# pyscopg2 is not compatible with PyPy
-lxml==4.1.1; platform_python_implementation!='CPython'
\ No newline at end of file
+lxml==4.1.1; platform_python_implementation!='CPython'
```
Comment (jerith) [inline requirements-debian.txt:9]: Did the `lxml` line change at all? Maybe a newline added/removed?

**Review by jerith — APPROVED** [review]
:+1:

File: requirements-debian.txt (hunk 1 above)
Comment (JayH5) [reply requirements-debian.txt:9]: Trailing newline added.

**Pull request merged**